### PR TITLE
update home page and privacy page content

### PR DESCRIPTION
**Pull request by marcuschui2022**
looks good!

#home page content change
#privacy content change

### CODE REVIEW

File: MyCoreApp/Pages/Index.cshtml [hunk 1]
```diff
@@ -5,6 +5,6 @@
 }
 
 <div class="text-center">
-    <h1 class="display-4">Welcome</h1>
+    <h1 class="display-4">Hello Kwok Yuk Chui</h1>
```
Comment (chloelam123) [inline MyCoreApp/Pages/Index.cshtml]: last name should be use all capital.

**Review by chloelam123 — CHANGES_REQUESTED** [review]
last name should be use all capital.

File: MyCoreApp/Pages/Index.cshtml (hunk 1 above)
Comment (marcuschui2022) [reply MyCoreApp/Pages/Index.cshtml]: got it!

**Review by chloelam123 — APPROVED** [review]
good job!

**Comment by chloelam123** [comment]
This PR closes #1 and resolves #2 .

**Pull request merged**